packages/tools/client-debugger/client-debugger-view/src/components/data-object-views/SharedMapView.tsx

```diff
@@ -44,28 +48,43 @@ export function SharedMapView(props: SharedMapViewProps): React.ReactElement {
 		};
 	}, [sharedMap, setEntries]);
 
+	const toggleCollapse = (key: string): void => {
+		setCollapsedEntries({
+			...collapsedEntries,
+			[key]: !collapsedEntries[key],
+		});
+	};
+
 	return (
-		<table style={{ borderCollapse: "collapse", width: "100%" }}>
-			<thead>
-				<tr>
-					<th>Key</th>
-					<th>Value</th>
-				</tr>
-			</thead>
-			<tbody style={{ borderCollapse: "collapse" }}>
-				{entries.map(([key, value]) => (
-					<tr key={key} style={{ borderCollapse: "collapse", border: "thin solid" }}>
-						<td
-							data-label="Key"
-							style={{ borderCollapse: "collapse", border: "thin solid" }}
-						>
-							{key}
-						</td>
-						<td data-label="Value">{getTableValue(value, renderChild)}</td>
-					</tr>
-				))}
-			</tbody>
-		</table>
+		<div>
+			{entries.map(([key, value]) => (
+				<div key={key}>
```

Comment:
It might be worth using the `Stack` / `StackItem` components from FluentUI and use their styling mechanisms instead of inline styling / CSS. Long term I would like to consider using something other than FluentUI, as I have been pretty unimpressed with it so far. But for now, that would give us a faster route to consistent styling across components.